ci/casper_updater/src/regex_data.rs

```diff
@@ -1,19 +1,26 @@
 #![allow(clippy::wildcard_imports)]
 
-use lazy_static::lazy_static;
+use once_cell::sync::Lazy;
 use regex::Regex;
 
 use crate::dependent_file::DependentFile;
 
-lazy_static! {
-    pub static ref MANIFEST_NAME_REGEX: Regex = Regex::new(r#"(?m)(^name = )"([^"]+)"#).unwrap();
-    pub static ref MANIFEST_VERSION_REGEX: Regex =
-        Regex::new(r#"(?m)(^version = )"([^"]+)"#).unwrap();
-    pub static ref PACKAGE_JSON_NAME_REGEX: Regex =
-        Regex::new(r#"(?m)(^  "name": )"([^"]+)"#).unwrap();
-    pub static ref PACKAGE_JSON_VERSION_REGEX: Regex =
-        Regex::new(r#"(?m)(^  "version": )"([^"]+)"#).unwrap();
-}
+pub static MANIFEST_NAME_REGEX: Lazy<Regex> = Lazy::new(|| {
+    let regex = Regex::new(r#"(?m)(^name = )"([^"]+)"#);
+    regex.unwrap()
+});
```

Comment:
There are a lot of new temporary variables throughout this PR. I personally feel like we should stay with inline values, and only pull variables out when it is very unclear what they stand for.

Reply:
I agree too, this was a hacky solution I came up with to prevent `cargo fmt` from blasting away my closures. I don't know a better way.

Reply:
I think it's fine if `cargo fmt` changes the layout.  I agree with @dwerner that all these temporaries should be removed.

Reply:
The temporary variables have been removed